templates/home/_fsf_moos.html

```diff
@@ -2,16 +2,46 @@
   <div class='col-6'>
     <h4>Why are snaps good for MOOS projects?</h4>
     <ul>
-      <li>Snaps are easy to discover and install. Millions of users can browse and install snaps graphically in the Snap Store or from the command-line.</li>
-      <li>Snaps install and run the same across Linux. They bundle the exact version of MOOS/MOOS-IvP required and your app’s library dependencies.</li>
-      <li>Snaps automatically update to the latest version. Four times a day, users’ systems will check for new versions and upgrade in the background.</li>
-      <li>Upgrades are not disruptive. Because upgrades are not in-place, users can keep your app open as it’s upgraded in the background.</li>
-      <li>Upgrades are safe. If your app fails to upgrade, users automatically roll back to the previous revision.</li>
+      <li>Bundle all the runtime requirements, including the exact version of MOOS/MOOS-IvP and system libraries you need.</li>
+      <li>Expand the distributions supported beyond just Ubuntu.</li>
+      <li>Directly and reliably control the delivery of application updates using existing infrastructure.</li>
```

Comment:
Can we reword this slightly (or add a new li) that mentions how safe upgrades are (rollbacks etc., like the last li being removed).

Reply:
Yes, I will re-add it.

Reply:
Still outstanding.